packages/server/src/drizzle/landing-page/template.dto.ts

```
@@ -0,0 +1,30 @@
+import { eq } from "drizzle-orm";
+import { DBConnection } from "../../drizzle/drizzle-client";
+import {
+	CreateTemplateSchema,
+	SelectTemplateSchema,
+	templateTable,
+} from "./template";
+
+export class TemplateTable extends DBConnection {
+	constructor(TURSO_CONNECTION_URL: string, TURSO_AUTH_TOKEN: string) {
+		super(TURSO_CONNECTION_URL, TURSO_AUTH_TOKEN);
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)